### ISSUE TITLE
Navigator seems to ignore ansible.cfg privilege escalation settings when running playbooks

### ISSUE THREAD

**Issue by mweetman-redhat** [ansible/ansible-navigator#297]
Tasks such as starting/stopping services or installing/removing packages fail when become|become_user|become_method are only defined in ansible.cfg. Running the same play from the cli using `ansible-playbook` works as expected.

**Comment by cidrblock** [comment]
@mweetman-redhat thx for checking it out

A couple questions:

1) Does it work with --ee True vs --ee false?
2) Can you include a fresh debug log here? (--ll debug --la False) So we can see what's going on?
3) Do the become_* values show in :config?

This is a good find, TY again

**Comment by mweetman-redhat** [comment]
Thanks for responding @cidrblock .. 
1. I was only using `--execution-environment false` as I haven't set up podman + EE.
2. Sorry, it looks like my env isn't reachable at the moment, I'll get someone to take a look and post debug output when I can
3. Yes, they do. `:config` seems to work as expected, it's just that the values aren't used by `:run`

**Comment by mweetman-redhat** [comment]
```210519133501.133 INFO 'ansible_navigator.setup_logger' New ansible-navigator instance, logging initialized
210519133501.133 DEBUG 'ansible_navigator.main' Share directory not found (development path)
210519133501.133 DEBUG 'ansible_navigator.main' Share directory not found (userbase)
210519133501.133 DEBUG 'ansible_navigator.main' Share directory found (sys.prefix)
210519133501.133 DEBUG 'ansible_navigator.main' Default collection_doc_cache_path set to: /home/student/.cache/ansible-navigator/collection_doc_cache.db
210519133501.133 DEBUG 'ansible_navigator.main' EDITOR environment variable not set
210519133501.133 DEBUG 'ansible_navigator.main' Default editor_command set to: vi +{line_number} {filename}
210519133501.133 DEBUG 'ansible_navigator.main' No config file set by ANSIBLE_NAVIGATOR_CONFIG
210519133501.133 DEBUG 'ansible_navigator.main' Skipping .ansible-navigator/ansible-navigator.yml because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping .ansible-navigator/ansible-navigator.yaml because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping .ansible-navigator/ansible-navigator.json because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /home/student/ansible-latest-complete/lib64/python3.6/site-packages/ansible_navigator/../etc/ansible-navigator/ansible-navigator.yml because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /home/student/ansible-latest-complete/lib64/python3.6/site-packages/ansible_navigator/../etc/ansible-navigator/ansible-navigator.yaml because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /home/student/ansible-latest-complete/lib64/python3.6/site-packages/ansible_navigator/../etc/ansible-navigator/ansible-navigator.json because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /home/student/.config/ansible-navigator/ansible-navigator.yml because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /home/student/.config/ansible-navigator/ansible-navigator.yaml because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /home/student/.config/ansible-navigator/ansible-navigator.json because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /etc/ansible-navigator/ansible-navigator.yml because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /etc/ansible-navigator/ansible-navigator.yaml because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /etc/ansible-navigator/ansible-navigator.json because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /usr/local/etc/ansible-navigator/ansible-navigator.yml because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /usr/local/etc/ansible-navigator/ansible-navigator.yaml because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' Skipping /usr/local/etc/ansible-navigator/ansible-navigator.json because it does not exist
210519133501.133 DEBUG 'ansible_navigator.main' No valid config file found, using all default values for configuration.
210519133501.133 DEBUG 'ansible_navigator.main' Command provided: '--execution-environment false run playbook-basic/start.yml -i inventory/inventory --ll debug --la False'
210519133501.133 DEBUG 'ansible_navigator.main' Completed post processing for cmdline. (changed=False)
210519133501.133 DEBUG 'ansible_navigator.main' Completed post processing for collection_doc_cache_path. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for editor_console. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' ansible-playbook found at /home/student/ansible-latest-complete/bin/ansible-playbook
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for execution_environment. (changed=True)
210519133501.134 DEBUG 'ansible_navigator.main'  before: 'false'
210519133501.134 DEBUG 'ansible_navigator.main'  after: 'False'
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for help_config. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for help_doc. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for inventory. (changed=True)
210519133501.134 DEBUG 'ansible_navigator.main'  before: '[['inventory/inventory']]'
210519133501.134 DEBUG 'ansible_navigator.main'  after: '['/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/inventory/inventory']'
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for inventory_column. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for log_append. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for log_file. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for mode. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for osc4. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for pass_environment_variable. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for playbook. (changed=True)
210519133501.134 DEBUG 'ansible_navigator.main'  before: 'playbook-basic/start.yml'
210519133501.134 DEBUG 'ansible_navigator.main'  after: '/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/playbook-basic/start.yml'
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for playbook_artifact_enable. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for playbook_artifact_replay. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for plugin_name. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Completed post processing for set_environment_variable. (changed=False)
210519133501.134 DEBUG 'ansible_navigator.main' Collection doc cache not mounted
210519133501.134 DEBUG 'ansible_navigator.main' Collection doc cache: 'path' is '/home/student/.cache/ansible-navigator/collection_doc_cache.db'
210519133501.134 DEBUG 'ansible_navigator.main' Collection doc cache: 'current version' is '0.6.2'
210519133501.134 DEBUG 'ansible_navigator.main' Running with app as 'run' (str/cli parameters)
210519133501.134 DEBUG 'ansible_navigator.main' Running with cmdline as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.134 DEBUG 'ansible_navigator.main' Running with collection_doc_cache_path as '/home/student/.cache/ansible-navigator/collection_doc_cache.db' (str/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with container_engine as 'podman' (str/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with editor_command as 'vi +{line_number} {filename}' (str/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with editor_console as 'True' (bool/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with execution_environment as 'False' (bool/cli parameters)
210519133501.134 DEBUG 'ansible_navigator.main' Running with execution_environment_image as 'quay.io/ansible/ansible-runner:devel' (str/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with help_config as 'False' (bool/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with help_doc as 'False' (bool/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with inventory as '['/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/inventory/inventory']' (list/cli parameters)
210519133501.134 DEBUG 'ansible_navigator.main' Running with inventory_column as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.134 DEBUG 'ansible_navigator.main' Running with log_append as 'False' (bool/cli parameters)
210519133501.134 DEBUG 'ansible_navigator.main' Running with log_file as '/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/ansible-navigator.log' (str/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with log_level as 'debug' (str/cli parameters)
210519133501.134 DEBUG 'ansible_navigator.main' Running with mode as 'interactive' (str/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with osc4 as 'True' (bool/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with pass_environment_variable as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.134 DEBUG 'ansible_navigator.main' Running with playbook as '/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/playbook-basic/start.yml' (str/cli parameters)
210519133501.134 DEBUG 'ansible_navigator.main' Running with playbook_artifact_enable as 'True' (bool/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with playbook_artifact_replay as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.134 DEBUG 'ansible_navigator.main' Running with playbook_artifact_save_as as '{playbook_dir}/{playbook_name}-artifact-{ts_utc}.json' (str/default configuration value)
210519133501.134 DEBUG 'ansible_navigator.main' Running with plugin_name as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.134 DEBUG 'ansible_navigator.main' Running with plugin_type as 'module' (str/default configuration value)
210519133501.135 DEBUG 'ansible_navigator.main' Running with set_environment_variable as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.144 DEBUG 'ansible_navigator.ui_framework.ui._set_colors' curses.COLORS: 256
210519133501.144 DEBUG 'ansible_navigator.ui_framework.ui._set_colors' curses.can_change_color: True
210519133501.145 DEBUG 'ansible_navigator.ui_framework.ui._set_colors' self._osc4: True
210519133501.145 DEBUG 'ansible_navigator.ui_framework.ui._set_colors' _custom_colors_enabled: True
210519133501.146 DEBUG 'ansible_navigator.ui_framework.ui._set_colors' Custom colors set
210519133501.179 DEBUG 'ansible_navigator.actions.run.run' run requested in interactive mode
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' No config file set by ANSIBLE_NAVIGATOR_CONFIG
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping .ansible-navigator/ansible-navigator.yml because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping .ansible-navigator/ansible-navigator.yaml because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping .ansible-navigator/ansible-navigator.json because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /home/student/ansible-latest-complete/lib64/python3.6/site-packages/ansible_navigator/../etc/ansible-navigator/ansible-navigator.yml because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /home/student/ansible-latest-complete/lib64/python3.6/site-packages/ansible_navigator/../etc/ansible-navigator/ansible-navigator.yaml because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /home/student/ansible-latest-complete/lib64/python3.6/site-packages/ansible_navigator/../etc/ansible-navigator/ansible-navigator.json because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /home/student/.config/ansible-navigator/ansible-navigator.yml because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /home/student/.config/ansible-navigator/ansible-navigator.yaml because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /home/student/.config/ansible-navigator/ansible-navigator.json because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /etc/ansible-navigator/ansible-navigator.yml because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /etc/ansible-navigator/ansible-navigator.yaml because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /etc/ansible-navigator/ansible-navigator.json because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /usr/local/etc/ansible-navigator/ansible-navigator.yml because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /usr/local/etc/ansible-navigator/ansible-navigator.yaml because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Skipping /usr/local/etc/ansible-navigator/ansible-navigator.json because it does not exist
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' No valid config file found, using all default values for configuration.
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Command provided: 'run'
210519133501.185 INFO 'ansible_navigator.actions.run_54a3._update_args' 'mode' cannot be reconfigured. (restore original)
210519133501.185 INFO 'ansible_navigator.actions.run_54a3._update_args' 'mode' cannot be reconfigured. (apply defaults)
210519133501.185 INFO 'ansible_navigator.actions.run_54a3._update_args' 'mode' cannot be reconfigured (apply previous cli)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for cmdline. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for collection_doc_cache_path. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for editor_console. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' ansible-playbook found at /home/student/ansible-latest-complete/bin/ansible-playbook
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for execution_environment. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for help_config. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for help_doc. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for inventory. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for inventory_column. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for log_append. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for log_file. (changed=False)
210519133501.185 INFO 'ansible_navigator.actions.run_54a3._update_args' 'mode' cannot be reconfigured. (post process)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for osc4. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for pass_environment_variable. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for playbook. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for playbook_artifact_enable. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for playbook_artifact_replay. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for plugin_name. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Completed post processing for set_environment_variable. (changed=False)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with app as 'run' (str/cli parameters)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with cmdline as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with collection_doc_cache_path as '/home/student/.cache/ansible-navigator/collection_doc_cache.db' (str/default configuration value)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with container_engine as 'podman' (str/default configuration value)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with editor_command as 'vi +{line_number} {filename}' (str/default configuration value)
210519133501.185 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with editor_console as 'True' (bool/default configuration value)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with execution_environment as 'False' (bool/previous cli command)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with execution_environment_image as 'quay.io/ansible/ansible-runner:devel' (str/default configuration value)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with help_config as 'False' (bool/default configuration value)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with help_doc as 'False' (bool/default configuration value)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with inventory as '['/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/inventory/inventory']' (list/previous cli command)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with inventory_column as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with log_append as 'False' (bool/previous cli command)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with log_file as '/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/ansible-navigator.log' (str/default configuration value)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with log_level as 'debug' (str/previous cli command)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with mode as 'interactive' (str/default configuration value)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with osc4 as 'True' (bool/default configuration value)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with pass_environment_variable as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with playbook as '/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/playbook-basic/start.yml' (str/previous cli command)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with playbook_artifact_enable as 'True' (bool/default configuration value)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with playbook_artifact_replay as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with playbook_artifact_save_as as '{playbook_dir}/{playbook_name}-artifact-{ts_utc}.json' (str/default configuration value)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with plugin_name as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with plugin_type as 'module' (str/default configuration value)
210519133501.186 DEBUG 'ansible_navigator.actions.run_54a3._update_args' Running with set_environment_variable as 'Constants.NOT_SET' (Constants/value has not been set)
210519133501.186 DEBUG 'ansible_navigator.runner.api.generate_run_command_args' Runner arg: json_mode:True
210519133501.186 DEBUG 'ansible_navigator.runner.api.generate_run_command_args' Runner arg: quiet:True
210519133501.186 DEBUG 'ansible_navigator.runner.api.generate_run_command_args' Runner arg: cancel_callback:<bound method BaseRunner.runner_cancelled_callback of <ansible_navigator.runner.api.CommandRunnerAsync object at 0x7f78c3745ac8>>
210519133501.186 DEBUG 'ansible_navigator.runner.api.generate_run_command_args' Runner arg: finished_callback:<bound method BaseRunner.runner_finished_callback of <ansible_navigator.runner.api.CommandRunnerAsync object at 0x7f78c3745ac8>>
210519133501.186 DEBUG 'ansible_navigator.runner.api.generate_run_command_args' Runner arg: envvars:{'ANSIBLE_CONFIG': '/usr/local/etc/ansible-navigator/ansible.cfg'}
210519133501.186 DEBUG 'ansible_navigator.runner.api.generate_run_command_args' Runner arg: cwd:/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/playbook-basic
210519133501.186 DEBUG 'ansible_navigator.runner.api.generate_run_command_args' Runner arg: executable_cmd:/home/student/ansible-latest-complete/bin/ansible-playbook
210519133501.186 DEBUG 'ansible_navigator.runner.api.generate_run_command_args' Runner arg: cmdline_args:['/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/playbook-basic/start.yml', '-i', '/home/student/Documents/RH294/classroom/grading/src/rh294/ansible/inventory/inventory']
210519133501.188 DEBUG 'ansible_navigator.actions.run_54a3._run_runner' runner requested to start
210519133501.188 INFO 'ansible_navigator.actions.run_54a3._init_run' Run initialized and playbook started.
210519133501.291 DEBUG 'ansible_navigator.actions.run_54a3._take_step' autoscroll enabled
210519133501.495 DEBUG 'ansible_navigator.actions.run_54a3._dequeue' Drained 5 events
210519133502.207 DEBUG 'ansible_navigator.actions.run_54a3._dequeue' Drained 2 events
210519133502.310 DEBUG 'ansible_navigator.actions.run_54a3._dequeue' Drained 8 events
210519133502.412 DEBUG 'ansible_navigator.actions.run_54a3._dequeue' Drained 1 events
210519133528.713 DEBUG 'ansible_navigator.actions.run_54a3._dequeue' Drained 3 events
210519133528.918 DEBUG 'ansible_navigator.actions.run_54a3.update' runner finished
210519133528.918 INFO 'ansible_navigator.actions.run_54a3.update' Playbook complete
210519133528.918 DEBUG 'ansible_navigator.actions.run_54a3.write_artifact' Formatted artifact file name set to /home/student/Documents/RH294/classroom/grading/src/rh294/ansible/playbook-basic/start-artifact-2021-05-19T13:35:28.918657+00:00.json
210519133528.918 DEBUG 'ansible_navigator.actions.run_54a3.write_artifact' Resolved artifact file name set to /home/student/Documents/RH294/classroom/grading/src/rh294/ansible/playbook-basic/start-artifact-2021-05-19T13:35:28.918657+00:00.json
210519133528.919 INFO 'ansible_navigator.actions.run_54a3.write_artifact' Saved artifact as /home/student/Documents/RH294/classroom/grading/src/rh294/ansible/playbook-basic/start-artifact-2021-05-19T13:35:28.918657+00:00.json
210519133603.407 DEBUG 'ansible_navigator.actions.select.run' selection made
210519133603.408 DEBUG 'ansible_navigator.actions.select.run' Requested next step in run_54a3 will be task_list
210519133606.671 DEBUG 'ansible_navigator.actions.select.run' selection made
210519133606.671 DEBUG 'ansible_navigator.actions.select.run' Requested next step in run_54a3 will be task
210519133606.687 DEBUG 'ansible_navigator.ui_framework.ui._color_lines_for_term' Added color: 16:(807, 568, 470)
210519133606.687 DEBUG 'ansible_navigator.ui_framework.ui._color_lines_for_term' Added color: 17:(709, 807, 658)
210519133606.687 DEBUG 'ansible_navigator.ui_framework.ui._color_lines_for_term' Added color: 18:(337, 611, 839)
210519133613.088 DEBUG 'ansible_navigator.actions.back.run' back requested
210519133613.088 DEBUG 'ansible_navigator.actions.back.run' Stepping back in run_54a3 from task to task_list
210519133613.960 DEBUG 'ansible_navigator.actions.back.run' back requested
210519133613.960 DEBUG 'ansible_navigator.actions.back.run' Stepping back in run_54a3 from task_list to plays
210519133614.983 DEBUG 'ansible_navigator.actions.back.run' back requested
210519133614.983 DEBUG 'ansible_navigator.actions.back.run' Return to run_54a3, at last step
210519133614.983 DEBUG 'ansible_navigator.actions.run_54a3.run' No steps remaining for 'run_54a3' returning to calling app
```

**Comment by mweetman-redhat** [comment]
@cidrblock  .. I'm guessing `No config file set by ANSIBLE_NAVIGATOR_CONFIG` is probably important?

**Comment by cidrblock** [comment]
@mweetman-redhat that message is about the navigator settings file, not the ansible.cfg,

what really interesting is that the ANSIBLE_CONFIG envar is being passed off:

```210519133501.186 DEBUG 'ansible_navigator.runner.api.generate_run_command_args' Runner arg: envvars:{'ANSIBLE_CONFIG': '/usr/local/etc/ansible-navigator/ansible.cfg'}
```
will need to try and recreate this locally, it might be a day or so.

thx again

**Comment by cidrblock** [comment]
This should have been fixed with #383, mind trying again?  pypi won't be updated until 6/1 so if you don't mind waiting until then or use source.

Thanks again for the testing & use!

**Comment by mweetman-redhat** [comment]
@cidrblock  .. many thanks, works perfectly now
